<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StampCircleDialog</class>
 <widget class="QDialog" name="StampCircleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>887</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>630</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Circle Stamp Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="bottomMargin">
      <number>7</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="circleStampLayoutLabel">
       <property name="styleSheet">
        <string notr="true">QLabel{color:rgb(1, 162, 254)}</string>
       </property>
       <property name="text">
        <string>Circle Stamp Layout</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="circleStampLayoutComboBox">
       <property name="styleSheet">
        <string notr="true">QComboBox QAbstractItemView::item {margin:2px;}</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="stampIntaglioCheckBox">
       <property name="text">
        <string>Intaglio</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="CircleGroupBox">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox#CircleGroupBox{border: 1px solid rgb(180, 180, 180)}
QGroupBox::title{color:rgb(25, 184, 250)}</string>
     </property>
     <property name="title">
      <string>Circle Property</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>11</number>
      </property>
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="rightMargin">
       <number>11</number>
      </property>
      <property name="bottomMargin">
       <number>11</number>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="labelAlignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="formAlignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="verticalSpacing">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="circleDiameterLabel">
          <property name="text">
           <string>Diameter</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="LaserDoubleSpinBox" name="circleDiameterSpinBox">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>0</height>
           </size>
          </property>
          <property name="suffix">
           <string>mm</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="circleBorderLabel">
          <property name="text">
           <string>Border</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="LaserDoubleSpinBox" name="circleBorderSpinBox">
          <property name="suffix">
           <string>mm</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="angleLabel">
          <property name="text">
           <string>Angle</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="angleSpinBox">
          <property name="maximum">
           <double>360.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="circleHeightLabel">
          <property name="text">
           <string>Height</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="circleHeightSpinBox">
          <property name="prefix">
           <string/>
          </property>
          <property name="suffix">
           <string>mm</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="innerCircleGroupBox">
        <property name="title">
         <string>Inner Circle</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QFormLayout" name="innerCircleLayout">
           <property name="labelAlignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="innerCircleMarginLabel">
             <property name="text">
              <string>Inner Circle Margins</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="LaserDoubleSpinBox" name="InnerCircleMarginDoubleSpinBox">
             <property name="suffix">
              <string>mm</string>
             </property>
             <property name="decimals">
              <number>3</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="innerCircleBorderLabel">
             <property name="text">
              <string>Inner Circle Border</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="LaserDoubleSpinBox" name="InnerCircleBorderDoubleSpinBox">
             <property name="suffix">
              <string>mm</string>
             </property>
             <property name="decimals">
              <number>3</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="textPropertyGroupBox">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox#textPropertyGroupBox{border: 1px solid rgb(180, 180, 180)}
QGroupBox::title{color:rgb(25, 184, 250)}</string>
     </property>
     <property name="title">
      <string>Text Property</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <property name="horizontalSpacing">
       <number>10</number>
      </property>
      <property name="verticalSpacing">
       <number>5</number>
      </property>
      <item row="10" column="0">
       <layout class="QFormLayout" name="formLayout_3">
        <property name="labelAlignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="formAlignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="horizontalSpacing">
         <number>10</number>
        </property>
        <property name="verticalSpacing">
         <number>11</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item row="4" column="0">
         <widget class="QLabel" name="hSpacelabel">
          <property name="text">
           <string>Text Spacing</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="LaserDoubleSpinBox" name="hSpaceSpinBox">
          <property name="suffix">
           <string>mm</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="marginLabel">
          <property name="text">
           <string>Text Margins</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="LaserDoubleSpinBox" name="marginSpinBox">
          <property name="suffix">
           <string>mm</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>98.989999999999995</double>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="textSizeLable">
          <property name="text">
           <string>Text Height</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="LaserDoubleSpinBox" name="textSizeSpinBox">
          <property name="prefix">
           <string/>
          </property>
          <property name="suffix">
           <string>mm</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QToolButton" name="boldBtn">
          <property name="text">
           <string>Bold</string>
          </property>
          <property name="icon">
           <iconset resource="../res/icons.qrc">
            <normaloff>:/ui/icons/images/bold.png</normaloff>:/ui/icons/images/bold.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="italicBtn">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../res/icons.qrc">
            <normaloff>:/ui/icons/images/straight.png</normaloff>:/ui/icons/images/straight.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFontComboBox" name="fontComboBox">
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="8" column="0">
       <widget class="QLineEdit" name="lineEdit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maxLength">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QTableView" name="tableView">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>188</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>188</height>
         </size>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="AdditionGroupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox#AdditionGroupBox{border: 1px solid rgb(180, 180, 180)}
QGroupBox::title{color:rgb(25, 184, 250)}</string>
     </property>
     <property name="title">
      <string>Addition Mark</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="labelAlignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="emblemLabel">
          <property name="text">
           <string>Emblem</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="emblemComboBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="emblemSizeLabel">
          <property name="text">
           <string>Emblem Size</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="embleSizeSpinBox">
          <property name="suffix">
           <string>mm</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="lineGroupBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Mark Line</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QFormLayout" name="formLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="lineWidthlabel">
             <property name="text">
              <string>Line Border</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="lineWidthSpinBox"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LaserDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>widget/LaserDoubleSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StampCircleDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StampCircleDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
